docs/all-resources-must-have-delete.md

```diff
@@ -18,15 +18,15 @@ The resource {resourceName} does not have a corresponding delete operation.
 
 ## Description
 
-All top level proxy and (tracked at all levels) resources MUST support delete.
+All proxy and tracked resources MUST support delete.
```

Comment:
>All proxy and tracked resources MUST support delete [](http://example.com/codeflow?start=0&length=51)

All tracked resources MUST support delete

Reply:
The other doc for RPC-Delete-V1-05 should say 

All proxy resources should support delete